lib/ZwaveClient.ts

```diff
@@ -65,6 +65,9 @@ import {
 	SerialAPISetupCommand,
 	FirmwareUpdateFileInfo,
 	FirmwareUpdateInfo,
```

Comment:
You got a linter warning here

Reply:
'FirmwareUpdateInfo' is defined but never used

Reply:
fixed